### PR TITLE
Adding kubernetes support

### PR DESCRIPTION
**Pull request by giovannicuriel**
This PR adds Kubernetes support for flowbroker.

This fixes dojot/dojot#500

### CODE REVIEW

File: orchestrator/nodeManager.js [hunk 1]
```diff
@@ -17,182 +11,22 @@ var template = require('./nodes/template/index').Handler;
...
-    });
-  }
-}
+var config = require("./config");
 
 class NodeManager {
   constructor() {
-    this.nodes = {
+    this.nodes = {};
+    this.addTenant("admin");
```
Comment (ericbaum) [inline orchestrator/nodeManager.js]: Why are you adding the "admin" tenant

File: orchestrator/nodes/dockerComposeRemoteNode/index.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+class DataHandler extends RemoteNode {
+  constructor(image, id) {
+    super();
+    this.info = {
+      userid: id,
+      image: image,
+      enabled: false
+    };
+
+    // TODO: socket path should be configurable
```
Comment (ericbaum) [inline orchestrator/nodes/dockerComposeRemoteNode/index.js]: Just add an environment variable for this part with a default value. No reason to delay this task

File: orchestrator/nodeManager.js (hunk 1 above)
Comment (giovannicuriel) [reply orchestrator/nodeManager.js]: No need for that.

File: orchestrator/nodes/dockerComposeRemoteNode/index.js (hunk 2 above)
Comment (giovannicuriel) [reply orchestrator/nodes/dockerComposeRemoteNode/index.js]: Ok

**Pull request merged**